### PR TITLE
[docs] Replace runwayml/stable-diffusion-v1-5 with Lykon/dreamshaper-8

### PR DESCRIPTION
**Pull request by suzukimain**
# What does this PR do?

`runwayml/stable-diffusion-v1-5` has been removed from huggingface and replaced with `Lykon/dreamshaper-8`.
Related to #9322 and #9323.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@stevhliu @sayakpaul

### CODE REVIEW

File: docs/source/en/using-diffusers/inpaint.md [hunk 1]
```diff
@@ -216,20 +216,20 @@ make_image_grid([init_image, mask_image, image], rows=1, cols=3)
 
 ## Non-inpaint specific checkpoints
 
-So far, this guide has used inpaint specific checkpoints such as [runwayml/stable-diffusion-inpainting](https://huggingface.co/runwayml/stable-diffusion-inpainting). But you can also use regular checkpoints like [runwayml/stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5). Let's compare the results of the two checkpoints.
+So far, this guide has used inpaint specific checkpoints such as [runwayml/stable-diffusion-inpainting](https://huggingface.co/runwayml/stable-diffusion-inpainting). But you can also use regular checkpoints like [Lykon/dreamshaper-8](https://huggingface.co/Lykon/dreamshaper-8). Let's compare the results of the two checkpoints.
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/inpaint.md:219]: Let's also update the inpainting checkpoint

```suggestion
So far, this guide has used inpaint specific checkpoints such as [Lykon/dreamshaper-8-inpainting](Lykon/dreamshaper-8-inpainting). But you can also use regular checkpoints like [Lykon/dreamshaper-8](https://huggingface.co/Lykon/dreamshaper-8). Let's compare the results of the two checkpoints.
```

**Review by stevhliu — APPROVED** [review]
Thanks a ton for the update!

File: docs/source/en/using-diffusers/inpaint.md (hunk 1 above)
Comment (suzukimain) [reply docs/source/en/using-diffusers/inpaint.md]: hi, @stevhliu. Fixed

Comment (julien-c) [reply docs/source/en/using-diffusers/inpaint.md]: they also exist here https://huggingface.co/stable-diffusion-v1-5/stable-diffusion-v1-5 FWIW

Comment (julien-c) [reply docs/source/en/using-diffusers/inpaint.md]: for future reproducibility, cc @apolinario

Comment (stevhliu) [reply docs/source/en/using-diffusers/inpaint.md]: Actually, if you don't mind @suzukimain, can you replace `Lykon/dreamshaper-8` with the `stable-diffusion-v1-5/stable-diffusion-v1-5` checkpoints?

Comment (suzukimain) [reply docs/source/en/using-diffusers/inpaint.md]: hello, @stevhliu.  `Lykon/dreamshaper-8` was replaced by `stable-diffusion-v1-5/stable-diffusion-v1-5`.
Also, please check `docs/source/en/using-diffusers/inpaint.md`

https://github.com/huggingface/diffusers/blob/302720ae3da8da74f3f150537035ce6057dee75f/docs/source/en/using-diffusers/inpaint.md?plain=1#L220

**Pull request merged**